frontend/projects/upgrade/src/app/features/dashboard/experiment-users/experiment-users-root/experiment-users-root.component.ts

```diff
@@ -9,13 +9,19 @@ import { StratificationFactorsService } from '../../../../core/stratification-fa
   styleUrls: ['./experiment-users-root.component.scss'],
 })
 export class ExperimentUsersRootComponent {
+  selectedTabIndex = 0;
   constructor(
     private experimentService: ExperimentService,
     private previewUsersService: PreviewUsersService,
     private stratificationFactorsService: StratificationFactorsService
   ) {}
 
+  ngOnInit() {
+    this.selectedTabChange({ index: this.selectedTabIndex });
```

Comment:
This will be called once at the start so it will always called with `this.selectedTabIndex = 0`. So do we need to make this a class variable?

Reply:
code updated